.reuse/dep5

```diff
@@ -101,8 +101,8 @@ Copyright: Material Icons <https://github.com/material-icons/material-icons/blob
 License: Apache-2.0
 
 Files: internal/compiler/widgets/cosmic-base/_*.svg
-Copyright: Material Icons <https://github.com/material-icons/material-icons/blob/master/LICENSE>
-License: Apache-2.0
+Copyright: Cosmic Icons <https://github.com/pop-os/cosmic-icons>
```

Comment:
The website says this:

When reusing this work please include a proper attribution:

> "[Cosmic Icons](http://github.com/pop-os/cosmic-icons)" by [System76](http://system76.com/) is licensed under [CC-SA-4.0](http://creativecommons.org/licenses/by-sa/4.0/)

So here in the copyright line, I think it should say

"[Cosmic Icons](http://github.com/pop-os/cosmic-icons)" by [System76](http://system76.com/)

Reply:
Ok done. Thanks